postpic/datahandling.py

```diff
@@ -297,7 +298,50 @@ def f(s, o):
     return ret
 
 
-class Field(object):
+def _reducing_numpy_method(method):
+    @functools.wraps(method)
```

Comment:
Please add a docstring to this function. Can you also add some comments in the code, **why** it is doing something or with what intention a codeblock was written?